src/Alpha_complex/test/Weighted_alpha_complex_unit_test.cpp

```diff
@@ -124,4 +124,48 @@ BOOST_AUTO_TEST_CASE(Weighted_alpha_complex_3d_comparison) {
     }
     ++dD_itr;
   }
-}
\ No newline at end of file
+}
+
+BOOST_AUTO_TEST_CASE(Is_weighted_alpha_complex_nan) {
+  using Kernel = CGAL::Epeck_d< CGAL::Dimension_tag<3> >;
+  using Bare_point = Kernel::Point_d;
+  using Weighted_point = Kernel::Weighted_point_d;
+  using Vector_of_points = std::vector<Weighted_point>;
+
+  Vector_of_points points;
+  points.emplace_back(Bare_point(1, -1, -1), 4.);
+  points.emplace_back(Bare_point(-1, 1, -1), 4.);
+  points.emplace_back(Bare_point(-1, -1, 1), 4.);
+  points.emplace_back(Bare_point(1, 1, 1),   4.);
+  points.emplace_back(Bare_point(2, 2, 2),   1.);
+
+  Gudhi::alpha_complex::Alpha_complex<Kernel, true> alpha_complex_from_weighted_points(points);
+
+  std::clog << "Weighted alpha complex\n";
+  Gudhi::Simplex_tree<> stree;
+  if (alpha_complex_from_weighted_points.create_complex(stree)) {
+    for (auto f_simplex : stree.filtration_simplex_range()) {
+      std::clog << "   ( ";
+      for (auto vertex : stree.simplex_vertex_range(f_simplex)) {
+        std::clog << vertex << " ";
+      }
+      std::clog << ") -> " << "[" << stree.filtration(f_simplex) << "]\n";
+
+      BOOST_CHECK(!std::isnan(stree.filtration(f_simplex)));
+    }
+  }
+  std::clog << "Weighted alpha complex with output_squared_values\n";
```

Comment:
Isn't it the other way around? "**without** output_squared_values", as `create_complex<false>` is used?

Reply:
I fixed the log on [a8d8634](https://github.com/GUDHI/gudhi-devel/pull/1172/commits/a8d86348b4e8fd161506b18e462872cd245c292b)